Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2756,12 +2759,12 @@ public List<PersonInfo> GetPeople(BaseItem item)
                 }
             }
 
-            return new List<PersonInfo>();
+            return [];
```

Comment:
Would `return ReadOnlyCollection<PersonInfo>.Empty;` be better here?

Reply:
its the same